subscriptions/api.go

```diff
@@ -0,0 +1,21 @@
+package subscriptions
+
+import (
+	zoho "github.com/schmorrison/Zoho"
+)
+
+const ZohoSubscriptionsOriganizationID = "X-com-zoho-subscriptions-organizationid"
```

Comment:
Change name to `ZohoSubscriptionsEndpointHeader` ala Invoice subpackage.

Reply:
Renamed to ZohoSubscriptionsEndpointHeader